src/lib/holocene/data-encoder-settings.svelte

```diff
@@ -52,23 +56,28 @@
   hideConfirm
   on:cancelModal={onCancel}
   on:confirmModal={onCancel}
+  cancelText="Done"
 >
-  <h3 slot="title" data-cy="data-encoder-title">Data Encoder</h3>
+  <h3 slot="title" data-cy="data-encoder-title" tabindex={0} autofocus>
```

Comment:
Ideally (instead of using `tabindex` and `autofocus` here) we might want to look into updating our Holocene modal component to take focus when opened and have a focus trap.

Reply:
I like that suggestion a lot can we make a Jira ticket for it and throw it in next sprint?

Reply:
Added ✅